extensions/wikia/TOC/js/modules/toc.mustache.js

```diff
@@ -0,0 +1,32 @@
+define('wikia.toc.mustache', function () {
+	'use strict';
+
+	return '{{#sections}}<li class="{{class}}">\
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use string concatenation rather than line continuation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AMultilineStringLiterals)